### ISSUE TITLE
Help-Items `Terms` and  `Privacy` should point elsewere when using a custom server

### ISSUE THREAD

**Issue by cmorty** [jitsi/jitsi-meet-electron#363]
When using a custom server the Help-Items `Terms` and `Privacy` should point to that server. Probably that will be a 404 for most instances, but that can IMO be fixed later.

**Comment by saghul** [comment]
Makes sense, but we don't currently have a goods way to know the link for those.

**Comment by cmorty** [comment]
@saghul Why not just set them `<URL>/meet/privacy` and `<URL>/meet/terms`. This seem to be the current URLs. If running your own server you can just redirect those to wherever you want. One might want extend the Docker accordingly, but IMO getting a 404 is still better than getting the wrong information.

**Comment by saghul** [comment]
That's just how we linked them, which is not even on the same domain, so that setup wouldn't work for meet.jit.si either.

Maybe a good start would be to disable those entries if the default server is not used.
